frontend/src/console/components/Hero/Hero.tsx

```diff
@@ -146,9 +162,15 @@ export class HeroComponent extends React.PureComponent<Props, State> {
     const field = e.target.name;
     const { value } = e.target;
 
-    this.setState({
-      [field]: value
-    });
+    if (field === 'title') {
+      this.setState({
+        title: value
+      });
+    } else if (field === 'subtitle') {
+      this.setState({
+        subtitle: value
+      });
+    }
```

Comment:
We don't need to rewrite it. It would be sufficient to do something like this:
```ts
this.setState({
  [field]: value
} as Pick<State, 'title' | 'subtitle'>);
```

Reply:
I learned this too, thanks @Agrendalath :smile: